### PR TITLE
feat: allow use of postgresql with gae

### PR DESCRIPTION
**Pull request by clakech**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by DanielFran** [comment]
@saturnism Can you please validate this one too please?

File: generators/gae/index.js [hunk 1]
```diff
@@ -555,8 +556,9 @@ module.exports = class extends BaseGenerator {
                 this.log(chalk.bold('\nCreating New Cloud SQL Instance'));
 
                 const name = this.gcpCloudSqlInstanceName;
+                const dbVersion = this.prodDatabaseType === 'postgresql' ? ' --database-version="POSTGRES_9_6" --tier="db-g1-small"' : '';
```
Comment (saturnism) [inline generators/gae/index.js]: for mysql, it uses the default tier. can we keep that the same for postgres?

also, would it be more clear to change `dbVersion` to `dbVersionFlag`.

Comment (clakech) [reply generators/gae/index.js]: We can't keep default tier for postgres, as described in doc. I tried but faced an explicit error then.

Yes dbVersionFlag may be better

Comment (clakech) [reply generators/gae/index.js]: done

**Comment by clakech** [comment]
Doc added https://github.com/jhipster/jhipster.github.io/pull/702

**Comment by DanielFran** [comment]
@saturnism Can you please confirm everything is OK to be merged? Thanks

**Comment by saturnism** [comment]
@DanielFran thnx for the updates!

**Comment by DanielFran** [comment]
Thanks @clakech

**Pull request merged**